lib/core/InterceptorManager.js

```diff
@@ -35,6 +35,15 @@ InterceptorManager.prototype.eject = function eject(id) {
   }
 };
 
+/**
+ * Clear all interceptors from the stack
+ */
+InterceptorManager.prototype.clear = function clear() {
+  if (this.handlers) {
+    this.handlers = null;
```

Comment:
Since the default value is `[]` and all methods are handled with `this.handlers` as an array I think you could set it as `[]` 🤔 

```js
this.handlers = [];
```

Reply:
agreed